contracts/ExchangeRates.sol

```diff
@@ -18,6 +18,9 @@ import "./interfaces/IExchanger.sol";
 
 
 // https://docs.synthetix.io/contracts/source/contracts/exchangerates
+// SIP-115:
+//   - Can we remove the internal rates?
```

Comment:
you mean https://sips.synthetix.io/sips/sip-81?

Reply:
Ah ha, yes, exactly. I noticed the current exchange rates contract had stopped being updated with https://sips.synthetix.io/sips/sip-95.

Since we'd have to deploy a new exchange rates contract with SIP-119 and thereby lose all the old state, it felt like a good time to do this as well.